src/quic/node_quic_buffer.cc

```diff
@@ -16,27 +16,27 @@ void QuicBufferChunk::MemoryInfo(MemoryTracker* tracker) const {
   tracker->TrackField("next", next_);
 }
 
-size_t QuicBuffer::push(uv_buf_t* bufs, size_t nbufs, DoneCB done) {
+size_t QuicBuffer::Push(uv_buf_t* bufs, size_t nbufs, DoneCB done) {
   size_t len = 0;
   if (nbufs == 0 || bufs == nullptr || is_empty(bufs[0])) {
-    done(0);
+    Done(0);
```

Comment:
This would call the wrong function now, wouldn’t it?

Reply:
good catch, bad search/replace.